src/ui/ui_menu_nbgl.c

```diff
@@ -101,7 +101,7 @@ static void displaySettingsMenu(void) {
 void ui_menu_main(void) {
     nbgl_useCaseHome("Stellar",
                      &C_icon_stellar_64px,
-                     "This app enables signing transactions\n on the Stellar network",
+                     NULL,
```

Comment:
Why ?

Reply:
`nbgl_useCaseHome` defaults to use case if the param is null